home_aerospace/main.py

```diff
@@ -0,0 +1,126 @@
+"""Default homescreen
+
+Hackedup awful code for a london aerospace themed badge
+"""
+
+___name___         = "Aerospace Badge"
+___license___      = "MIT"
+___categories___   = ["Homescreens"]
+___dependencies___ = ["homescreen", "wifi", "http", "ugfx_helper", "sleep"]
+___launchable___   = False
+___bootstrapped___ = True
```

Comment:
Please remove this line, as it's not a built-in app.

Reply:
good catch! I'll merge it and remove it manually.